back-end/src/database/migrations/20211110000658-create-users.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      role: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
```

Comment:
Aqui é válido a data já entrar automaticamente
```suggestion
        type: Sequelize.DATE,
        defaultValue: Sequelize.NOW
```